### DISCUSSION TITLE
索引能否按需调整和修改？

### DISCUSSION THREAD

**Discussion by jilieryuyi** [milvus-io/milvus#4931] [Q&A and General discussion] [answered]
比如一开始是这样的
param = {'nlist': 16384}
    # status = client.create_index(table_name, param)
    status = client.create_index(table_name, IndexType.IVF_FLAT, param)
    return status

调整为
param = {'nlist': 256}
    # status = client.create_index(table_name, param)
    status = client.create_index(table_name, IndexType.IVF_FLAT, param)
    return status

或者调整索引类型？是否是允许的，会不会有什么问题？

**Comment by shengjun1985** [comment, marked as answer]
It will drop the previous index and creates a new index.
